build-system/global-configs/experiments-const.json

```diff
@@ -1,5 +1,5 @@
 {
-  "INTERSECTION_OBSERVER_POLYFILL": false,
+  "INTERSECTION_OBSERVER_POLYFILL": true,
```

Comment:
wait did we accidentally ship the polyfill to inabox instead?

Reply:
hmmmm saw your comment in #28434

Reply:
I was told that it it didn't launch to inabox because config.json took precedence.

Reply:
Got it. Thanks
https://github.com/ampproject/amphtml/blob/b1cdabd46601c687549492c86b00afbd1ed3c711/build-system/babel-config/helpers.js#L79-L89